lntest/harness_assertion.go

```diff
@@ -255,10 +255,10 @@ func (h *HarnessTest) AssertNumActiveEdges(hn *node.HarnessNode,
 	// filterDisabled is a helper closure that filters out disabled
 	// channels.
 	filterDisabled := func(edge *lnrpc.ChannelEdge) bool {
-		if edge.Node1Policy.Disabled {
+		if edge.Node1Policy != nil && edge.Node1Policy.Disabled {
```

Comment:
hmm looking at this again, why would we ever have an `edge` that has no policy? Did you run into #7261 in the itest?

Reply:
> why would we ever have an edge that has no policy?

we get ChannelUpdates after ChannelAnnouncements. So we may have the announcement and not yet have any updates for it

Reply:
> Did you run into https://github.com/lightningnetwork/lnd/issues/7261 in the itest?

dont think so no. See the panics in the linked build

Reply:
also see other usages of these values  - we always nil-check them